src/AutoBuild.jl

```diff
@@ -728,6 +729,12 @@ here are the relevant actors, broken down in brief:
 
 * `require_license` enables a special audit pass that requires licenses to be
    installed by all packages.
+
+* `dont_dlopen`: don't try to `dlopen` library products. This is separate from
+   specifying `dont_dlopen` on a `LibraryProduct` in that it still results in
+   the generated JLL loading the library at run time, and only prevents
+   BinaryBuilder from doing so during JLL generation.
```

Comment:
If I understand correctly:
```suggestion
   BinaryBuilder from doing so during audit.
```

Reply:
No, it's not only during audit. `locate` gets called by `satisfied` which happens outside of the auditor (otherwise just setting `skip_audit` would have been sufficient.

Reply:
Oh, ugh, I see.